generators/chipyard/src/main/scala/config/AbstractConfig.scala

```diff
@@ -45,7 +45,6 @@ class AbstractConfig extends Config(
   new chipyard.config.WithNoSubsystemDrivenClocks ++             // drive the subsystem diplomatic clocks from ChipTop instead of using implicit clocks
   new chipyard.config.WithInheritBusFrequencyAssignments ++      // Unspecified clocks within a bus will receive the bus frequency if set
   new chipyard.config.WithPeripheryBusFrequencyAsDefault ++      // Unspecified frequencies with match the pbus frequency (which is always set)
-  new chipyard.config.WithMemoryBusFrequency(100.0) ++           // Default 100 MHz mbus
```

Comment:
Why was this necessary? Why not just use the PBUS frequency specified?

Reply:
We want to move towards explicitly specifying all frequencies. It's not obvious that pbus freq sets mbus freq when mbus freq is unspecified.

Reply:
In fact the behavior of pbus freq setting mbus freq is the result of a workaround on our side.

Reply:
This prevents the `DefaultClockFrequencyKey` being used for the MBUS... requiring an extra config. fragment to override  it (either a fragment to set the MBUS freq to the PBUS freq if the PBUS frequency changes or a fragment to reset the MBUS frequency to the `DefaultClockFrequencyKey`). I think we can just modify the comment stating that in this default setup the MBUS == PBUS frequency.

Reply:
The DefaultClockFrequencyKey was the workaround. In an ideal world, all clock sinks would explicitly set their clock frequencies.

If some config inheriting this wants to change the default mbus/pbus frequencies, it should do so explicitly, through using two config fragments, rather than relying on knowledge of default clock setting behaviors.

@davidbiancolin

Reply:
This also seems to break things that check the DTS frequency (since its an `Option`)... so anything that checks a DTS frequency would also have to check the `DefaultClockFrequencyKey` if the `Option` returns `None`.

Reply:
I reverted this.